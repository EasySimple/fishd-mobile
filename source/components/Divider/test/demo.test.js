import webDemoTest from '../../../tests/demoTest';

webDemoTest('Divider');
