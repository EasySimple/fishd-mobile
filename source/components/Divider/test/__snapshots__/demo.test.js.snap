// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Divider demo correctly 1`] = `
<div>
  <div
    class="components-divider-demo"
  >
    <div
      class="demo-title"
    >
      基本用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-divider fm-divider--hairline"
        role="separator"
      />
    </div>
  </div>
  <div
    class="components-divider-demo"
  >
    <div
      class="demo-title"
    >
      展示文本
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-divider fm-divider--hairline fm-divider--content-left"
        role="separator"
      >
        文本
      </div>
      <div
        class="fm-divider fm-divider--hairline fm-divider--content-right"
        role="separator"
      >
        文本
      </div>
    </div>
  </div>
  <div
    class="components-divider-demo"
  >
    <div
      class="demo-title"
    >
      Dashed
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-divider fm-divider--dashed fm-divider--hairline fm-divider--content-center"
        role="separator"
      >
        文本
      </div>
    </div>
  </div>
  <div
    class="components-divider-demo"
  >
    <div
      class="demo-title"
    >
      自定义样式
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-divider fm-divider--dashed fm-divider--hairline fm-divider--content-center"
        role="separator"
        style="color:#337EFF;border-color:#337EFF;padding:0 16px"
      >
        FishDesign Mobile
      </div>
    </div>
  </div>
</div>
`;
